docs/source/notes/architecture_overview.rst

```diff
@@ -103,21 +83,3 @@ Automatic Feature Engineering
 Automatic feature engineering is for users to find the best features for their tasks. A detailed description of automatic feature engineering and its usage can be found `here <FeatureEngineering/Overview.rst>`__. It is supported through NNI trial SDK, which means you do not have to create an NNI experiment. Instead, simply import a built-in auto-feature-engineering algorithm in your trial code and directly run your trial code. 
```

Comment:
is "`here <FeatureEngineering/Overview.rst>`__." this syntax still work?

Reply:
I find only `.html` can work, like `here <FeatureEngineering/Overview.html>`__.

Reply:
another style is like this 
![image](https://user-images.githubusercontent.com/16907603/155252586-65d75f72-cffe-4cce-a189-72a984fb7145.png)
@ultmaster we should provide guideline for it